docs/training/apis-howto/overview.rst

```diff
@@ -1,32 +1,33 @@
-###############
- Training APIs
-###############
+#####################
+ Training API Guides
+#####################
 
-This section describes how to use the training APIs and contains the API reference information.
+The Training API Guides describe how to use the Determined training APIs. 
+Each API guide contains a link to its corresponding API reference.
 
-Determined leverages specific APIs for each Deep Learning framework. In general, users convert their
+Determined leverages specific APIs for each Deep Learning framework. In general, you convert your
 existing training code by subclassing a Trial class and implementing methods that advertise
-components of the user's model - e.g., model architecture, data loader, optimizer, learning rate
-scheduler, callbacks, etc. This is called the Trial definition and by structuring their code in this
-way, Determined is able to run the training loop and provide advanced training and model management
+components of your model - e.g., model architecture, data loader, optimizer, learning rate
+scheduler, callbacks, etc. This is called the Trial definition. With the code structured in this way,
+Determined is able to run the training loop and provide advanced training and model management
 capabilities.
 
-Once users' model code are ported to Determined's APIs, they can use an :doc:`experiment
-configuration </reference/reference-training/experiment-config-reference>` to configure how
-Determined should train the model - e.g., multi-GPU, hyperparameter search, etc.
+Once your model code is ported to a Determined API, you can use an :doc:`experiment
```

Comment:
The word "port" also suggests a Trial API, because it implies there's quite a bit of work in going from your current model to a model that works with Determined.  That's not exactly true of Core API, and we'd like it to also not be required with the eventual high-level APIs we are planning on writing.

Reply:
i suggest changing **ported** to **converted**

e.g.,

Once your model code is converted, you can use ...